draft-irtf-cfrg-rsa-blind-signatures.md

```diff
@@ -740,75 +632,240 @@ This document makes no IANA requests.
 
 # Test Vectors
 
-This section includes test vectors for the blind signature protocol defined in {{internal}}.
-It does not include test vectors based on the external interface in {{salted-interface}}.
+This section includes test vectors for the blind signature protocol defined in {{core-protocol}}.
 The following parameters are specified for each test vector:
 
 - p, q, n, e, d: RSA private and public key parameters, each encoded as a hexadecimal string.
-- msg: Messsage being signed, encoded as a hexadecimal string. The hash is computed using SHA-384.
+- msg: Input messsage being signed, encoded as a hexadecimal string. The hash is computed using SHA-384.
+- msg\_salt: Message randomizer salt, encoded as a hexadecimal string. This is only present for variants
```

Comment:
One source of confusion to me is that there are two salts, and both are related to the message. 
One is used as part of the standard function EMSA_PSS_ENCODE, and its length is either 0 or 384. 
The other is just a prefix added by the Randomize(msg) function. I suggest to change the name of this variable: `random_prefix` may work.

Reply:
That's a good idea -- I changed this to msg_prefix.